Build failure: wivrn

### Nixpkgs version

- Unstable (25.05)

### Steps to reproduce

- Update flake.lock to the following

```
• Updated input 'nixpkgs':
    'github:nixos/nixpkgs/fab95ba4b9523f310644e6e6087c0014535c8e02' (2025-05-13)
  → 'github:nixos/nixpkgs/b965e4c283060415956ccd39eee4ca34a6a56cf8' (2025-05-16)
```
- Rebuild flake

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/297508440

### Relevant log output

```console
Sourcing auto-add-driver-runpath-hook
Using autoAddDriverRunpath
Sourcing fix-elf-files.sh
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/n302pcz6z1rsk08g6j5r963wk9vjg5xd-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DC>
-- The C compiler identification is GNU 14.2.1
-- The CXX compiler identification is GNU 14.2.1
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/0fsnicvfpf55nkza12cjnad0w84d6ba7-gcc-wrapper-14.2.1.20250322/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/0fsnicvfpf55nkza12cjnad0w84d6ba7-gcc-wrapper-14.2.1.20250322/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/2crk9xnq5x9v7yf0r2nwkgj8qsmxr4ly-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Found Vulkan: /nix/store/nszik1q8ffmvsqk54kbc75dwyxwvi2nm-vulkan-loader-1.4.313.0/lib/libvulkan.so (found version "1.4.313") found com>
-- Found Git: /nix/store/805a5wv1cyah5awij184yfad1ksmbh9f-git-2.49.0/bin/git (found version "2.49.0")
-- Found OpenSSL: /nix/store/8pviily4fgsl02ijm65binz236717wfs-openssl-3.4.1/lib/libcrypto.so (found version "3.4.1")
CMake Warning (dev) at CMakeLists.txt:86 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found Boost: /nix/store/ammv4hfx001g454rn0dlgibj1imn9rkw-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found suitable ver>
-- Checking for modules 'libavcodec;libavutil;libswscale;libavfilter'
--   Found libavcodec, version 61.19.101
--   Found libavutil, version 59.39.100
--   Found libswscale, version 8.3.100
--   Found libavfilter, version 10.4.100
-- Checking for module 'libdrm'
--   Found libdrm, version 2.4.124
-- Checking for module 'x264'
--   Found x264, version 0.164.x
-- Checking for module 'libsystemd'
--   Found libsystemd, version 257
-- Checking for module 'libpipewire-0.3'
--   Found libpipewire-0.3, version 1.4.2
-- Checking for module 'libpulse'
--   Found libpulse, version 17.0
-- Checking for modules 'avahi-client;avahi-glib'
--   Found avahi-client, version 0.8
--   Found avahi-glib, version 0.8
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found nlohmann_json: /nix/store/l6wng97amh2h2saa5dpvbx5gavjv95r4-nlohmann_json-3.11.3/share/cmake/nlohmann_json/nlohmann_jsonConfig.cm>
-- Checking for modules 'glib-2.0;gio-2.0;gio-unix-2.0'
--   Found glib-2.0, version 2.84.1
--   Found gio-2.0, version 2.84.1
--   Found gio-unix-2.0, version 2.84.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for module 'libnotify'
--   Found libnotify, version 0.8.6
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/z88avybj8n2svi9wv1hl937k2k3mbc2d-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/68z67micavkw4l1swqdcias3n0a6r06q-vulkan-headers-1.4.313.0/include
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Found OpenMP_CXX: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
-- Found KF6Kirigami: /nix/store/m25ypcq3q6kdvnm1mscq4d6nv8ac5df4-kirigami-6.14.0-dev/lib/cmake/KF6Kirigami/KF6KirigamiConfig.cmake (foun>
-- Found Gettext: /nix/store/sdqvwr8gc74ms9cgf56yvy409xvl8hsf-gettext-0.22.5/bin/msgmerge (found version "0.22.5")
-- Found KF6I18n: /nix/store/7gr44wsqv1y2hmbni09292zlr22pxcz0-ki18n-6.14.0-dev/lib/cmake/KF6I18n/KF6I18nConfig.cmake (found version "6.14>
-- Found KF6CoreAddons: /nix/store/i3zgwfs3153j3f4sfx645qqid9jxlfh5-kcoreaddons-6.14.0-dev/lib/cmake/KF6CoreAddons/KF6CoreAddonsConfig.cm>
-- Found KF6QQC2DesktopStyle: /nix/store/93rw7n6jc3wlh63sczd7i3nwklz4maly-qqc2-desktop-style-6.14.0-dev/lib/cmake/KF6QQC2DesktopStyle/KF6>
-- Found KF6IconThemes: /nix/store/blpz9vjxvi5c6fib0v2fmhzkhdv5xr63-kiconthemes-6.14.0-dev/lib/cmake/KF6IconThemes/KF6IconThemesConfig.cm>
-- Found KF6: success (found version "6.14.0") found components: Kirigami I18n CoreAddons QQC2DesktopStyle IconThemes
-- Found Python3: /nix/store/8w718rm43x7z73xhw9d6vh8s4snrq67h-python3-3.12.10/bin/python3.12 (found version "3.12.10") found components: >
-- Could NOT find HIDAPI (missing: HIDAPI_INCLUDE_DIR HIDAPI_LIBRARY)
-- Could NOT find bluetooth (missing: bluetooth_LIBRARY bluetooth_INCLUDE_DIR)
-- Could NOT find OpenHMD (missing: OPENHMD_INCLUDE_DIR OPENHMD_LIBRARY)
-- Could NOT find OpenCV (missing: OpenCV_DIR)
-- Could NOT find Libusb1 (missing: LIBUSB1_LIBRARY LIBUSB1_INCLUDE_DIR)
-- Found JPEG: /nix/store/9f6bvnw1hxy79shw6lva854ck3cmi43j-libjpeg-turbo-3.0.4/lib/libjpeg.so (found version "62")
-- Could NOT find realsense2 (missing: realsense2_DIR)
-- Could NOT find depthai (missing: depthai_DIR)
-- Could NOT find SDL2 (missing: SDL2_DIR)
-- Found ZLIB: /nix/store/srby6wmvg7dp454pwb6qvaxdiri38sc1-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Could NOT find cJSON (missing: CJSON_INCLUDE_DIR CJSON_LIBRARY)
-- Could NOT find LeapV2 (missing: LeapV2_INCLUDE_DIR LeapV2_LIBRARY)
-- Could NOT find LeapSDK (missing: LeapSDK_DIR)
-- Found ONNXRuntime: /nix/store/pbnli8102gvv4548b87349iikcnaflkc-onnxruntime-1.21.0-dev/include/onnxruntime
-- Performing Test HAVE_EGL
-- Performing Test HAVE_EGL - Success
-- Found EGL: /nix/store/y0c6p6cxq0nyh09lw0lpwps5i3dn1v6n-libglvnd-1.7.0-dev/include (found version "1.5")
-- Could NOT find Percetto (missing: Percetto_INCLUDE_DIR Percetto_LIBRARY)
-- Found Systemd: /nix/store/2vixc3z7hl2c1liq1624lrkw2lkc3hgm-systemd-257.5-dev/include
-- Found OpenGLES: /nix/store/z88avybj8n2svi9wv1hl937k2k3mbc2d-libglvnd-1.7.0/lib/libGLESv2.so  found components: V3
-- Checking for module 'libuvc'
--   No package 'libuvc' found
-- Found X11: /nix/store/18w6fpxmn5px02bpfgk702bs9k7yj5ml-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/mpvxc1dbpnk74345lk69dw497iqcjvj0-libX11-1.8.12/lib/libX11.so;/nix/store/gmirqf6vp6rskn2dhfyd7ha>
-- Looking for XOpenDisplay in /nix/store/mpvxc1dbpnk74345lk69dw497iqcjvj0-libX11-1.8.12/lib/libX11.so;/nix/store/gmirqf6vp6rskn2dhfyd7ha>
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Found udev: /nix/store/if9z6wmzmb07j63c02mvfkhn1mw1w5p4-systemd-257.5/lib/libudev.so
-- Checking for module 'xrandr'
--   Found xrandr, version 1.5.4
-- Checking for modules 'xcb;xcb-randr;x11-xcb'
--   Found xcb, version 1.17.0
--   Found xcb-randr, version 1.17.0
--   Found x11-xcb, version 1.8.12
-- Checking for one of the modules 'wayland-client'
-- Checking for one of the modules 'wayland-scanner'
-- Checking for one of the modules 'wayland-protocols'
-- Found OpenGL: /nix/store/z88avybj8n2svi9wv1hl937k2k3mbc2d-libglvnd-1.7.0/lib/libOpenGL.so  found components: GLX
-- Checking for one of the modules 'dbus-1'
-- Checking for one of the modules 'libbsd'
-- Checking for modules 'gstreamer-1.0;gstreamer-app-1.0;gstreamer-video-1.0'
--   No package 'gstreamer-1.0' found
--   No package 'gstreamer-app-1.0' found
--   No package 'gstreamer-video-1.0' found
-- Checking for module 'survive'
--   No package 'survive' found
-- OpenVR not found
-- Enabled drivers: drv_cemu drv_multi steamvr_lh v4l2
CMake Warning (dev) at /nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/cmake/GenerateKhrManifest.cmake:93 (add_custom_command):
  The following keywords are not supported when using
  add_custom_command(TARGET): DEPENDS.

  Policy CMP0175 is not set: add_custom_command() rejects invalid arguments.
  Run "cmake --help-policy CMP0175" for policy details.  Use the cmake_policy
  command to set the policy and suppress this warning.
Call Stack (most recent call first):
  /nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/cmake/GenerateOpenXRRuntimeManifest.cmake:76 (generate_khr_manifest_buildtre>
  /nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/targets/openxr/CMakeLists.txt:73 (generate_openxr_runtime_manifest_b>
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
-- #####----- Config -----#####
-- #    GIT_DESC:        v0.24.1
-- #
-- #    BLUETOOTH:       OFF
-- #    D3D11:           OFF
-- #    D3D12:           OFF
-- #    DBUS:            OFF
-- #    EGL:             ON
-- #    GST (GStreamer): OFF
-- #    HIDAPI:          OFF
-- #    JPEG:            OFF
-- #    LIBBSD:          OFF
-- #    LIBUSB:          OFF
-- #    LIBUVC:          OFF
-- #    ONNXRUNTIME:     OFF
-- #    OPENCV:          OFF
-- #    OPENGL:          ON
-- #    OPENGLES:        ON
-- #    OPENGL_GLX:      ON
-- #    PERCETTO:        OFF
-- #    REALSENSE:       OFF
-- #    OPENVR:          OFF
-- #    SDL2:            OFF
-- #    SYSTEM_CJSON:    OFF
-- #    SYSTEMD:         ON
-- #    TRACY:           OFF
-- #    VULKAN:          ON
-- #    WAYLAND:         OFF
-- #    WAYLAND_DIRECT:  OFF
-- #    XCB:             ON
-- #    XLIB:            ON
-- #    XRANDR:          ON
-- #
-- #    MODULE_AUX_VIVE:             ON
-- #    MODULE_COMPOSITOR:           ON
-- #    MODULE_COMPOSITOR_MAIN:      ON
-- #    MODULE_COMPOSITOR_NULL:      ON
-- #    MODULE_MERCURY_HANDTRACKING: OFF
-- #    MODULE_IPC:                  ON
-- #    MODULE_MONADO_GUI:           OFF
-- #    MODULE_MONADO_CLI:           ON
-- #
-- #    FEATURE_AHARDWARE_BUFFER:                      OFF
-- #    FEATURE_CLIENT_DEBUG_GUI:                      OFF
-- #    FEATURE_COLOR_LOG:                             ON
-- #    FEATURE_DEBUG_GUI:                             OFF
-- #    FEATURE_OPENXR:                                ON
-- #    FEATURE_OPENXR_ACTIVE_ACTION_SET_PRIORITY:     ON
-- #    FEATURE_OPENXR_BODY_TRACKING_FB:               OFF
-- #    FEATURE_OPENXR_FACE_TRACKING2_FB:              ON
-- #    FEATURE_OPENXR_DEBUG_UTILS:                    OFF
-- #    FEATURE_OPENXR_DISPLAY_REFRESH_RATE:           ON
-- #    FEATURE_OPENXR_FACIAL_TRACKING_HTC:            ON
-- #    FEATURE_OPENXR_FORCE_FEEDBACK_CURL:            ON
-- #    FEATURE_OPENXR_HAND_TRACKING_EXT:              ON
-- #    FEATURE_OPENXR_HEADLESS:                       ON
-- #    FEATURE_OPENXR_INTERACTION_EXT_EYE_GAZE:       ON
-- #    FEATURE_OPENXR_INTERACTION_EXT_HAND:           OFF
-- #    FEATURE_OPENXR_INTERACTION_EXT_PALM_POSE:      ON
-- #    FEATURE_OPENXR_INTERACTION_ML2:                ON
-- #    FEATURE_OPENXR_INTERACTION_MNDX:               ON
-- #    FEATURE_OPENXR_INTERACTION_MSFT_HAND:          OFF
-- #    FEATURE_OPENXR_INTERACTION_OPPO:               ON
-- #    FEATURE_OPENXR_INTERACTION_WINMR:              ON
-- #    FEATURE_OPENXR_LAYER_COLOR_SCALE_BIAS:         OFF
-- #    FEATURE_OPENXR_LAYER_CUBE:                     OFF
-- #    FEATURE_OPENXR_LAYER_CYLINDER:                 ON
-- #    FEATURE_OPENXR_LAYER_DEPTH:                    ON
-- #    FEATURE_OPENXR_LAYER_EQUIRECT1:                OFF
-- #    FEATURE_OPENXR_LAYER_EQUIRECT2:                ON
-- #    FEATURE_OPENXR_LAYER_FB_ALPHA_BLEND:           OFF
-- #    FEATURE_OPENXR_LAYER_FB_DEPTH_TEST:            OFF
-- #    FEATURE_OPENXR_LAYER_FB_IMAGE_LAYOUT           OFF
-- #    FEATURE_OPENXR_LAYER_FB_SETTINGS:              OFF
-- #    FEATURE_OPENXR_LAYER_FB_PASSTHROUGH:           OFF
-- #    FEATURE_OPENXR_OVERLAY:                        ON
-- #    FEATURE_OPENXR_PERFORMANCE_SETTINGS:           OFF
-- #    FEATURE_OPENXR_SPACE_LOCAL_FLOOR:              ON
-- #    FEATURE_OPENXR_SPACE_UNBOUNDED:                ON
-- #    FEATURE_OPENXR_VISIBILITY_MASK:                ON
-- #    FEATURE_OPENXR_VULKAN_SWAPCHAIN_FORMAT_LIST:   ON
-- #    FEATURE_OPENXR_XDEV_SPACE:                     ON
-- #    FEATURE_RENDERDOC:                             OFF
-- #    FEATURE_SERVICE:                               ON
-- #    FEATURE_SERVICE_SYSTEMD:                       OFF
-- #    FEATURE_SLAM:                                  OFF
-- #    FEATURE_SSE2:                                  ON
-- #    FEATURE_STEAMVR_PLUGIN:                        OFF
-- #    FEATURE_TRACING:                               OFF
-- #    FEATURE_WINDOW_PEEK:                           OFF
-- #
-- #    DRIVER_ANDROID:              OFF
-- #    DRIVER_ARDUINO:              OFF
-- #    DRIVER_DAYDREAM:             OFF
-- #    DRIVER_DEPTHAI:              OFF
-- #    DRIVER_EUROC:                OFF
-- #    DRIVER_HANDTRACKING:         OFF
-- #    DRIVER_HDK:                  OFF
-- #    DRIVER_HYDRA:                OFF
-- #    DRIVER_ILLIXR:               OFF
-- #    DRIVER_NS:                   OFF
-- #    DRIVER_OHMD:                 OFF
-- #    DRIVER_OPENGLOVES:           OFF
-- #    DRIVER_PSMV:                 OFF
-- #    DRIVER_PSSENSE:              OFF
-- #    DRIVER_PSVR:                 OFF
-- #    DRIVER_QWERTY:               OFF
-- #    DRIVER_REALSENSE:            OFF
-- #    DRIVER_REMOTE:               OFF
-- #    DRIVER_RIFT_S:               OFF
-- #    DRIVER_ROKID:                OFF
-- #    DRIVER_SIMULATED:            OFF
-- #    DRIVER_SIMULAVR:             OFF
-- #    DRIVER_SURVIVE:              OFF
-- #    DRIVER_TWRAP:                OFF
-- #    DRIVER_ULV2:                 OFF
-- #    DRIVER_ULV5:                 OFF
-- #    DRIVER_VF:                   OFF
-- #    DRIVER_VIVE:                 OFF
-- #    DRIVER_WMR:                  OFF
-- #    DRIVER_XREAL_AIR:            OFF
-- #    DRIVER_STEAMVR_LIGHTHOUSE:   TRUE
-- #####----- Config -----#####
Selected encoders:
        NVENC : TRUE
        VAAPI : TRUE
        Vulkan: TRUE
        x264  : TRUE

Audio backends:
        Pipewire  : TRUE
        Pulseaudio: TRUE

Optional features:
        systemd: ON
Installing in /nix/store/c3hy79zngd2cah9i5kcaf2794jsaxkpv-wivrn-0.24.1. Run /build/source/build/dashboard/prefix.sh to set the environmen>
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
CMake Warning at build/dashboard/Findorg.kde.kirigami-QMLModule.cmake:23 (message):
  qmldir not found in
  /nix/store/c3hy79zngd2cah9i5kcaf2794jsaxkpv-wivrn-0.24.1/lib/qt-6/qml/org/kde/kirigami
  or
  /nix/store/8qs879yx84gng0yljdybwi4fbbgg3dyy-qtbase-6.9.0/lib/qt-6/qml/org/kde/kirigami/qmldir
Call Stack (most recent call first):
  /nix/store/ki26gz0c85r02fx3mq1bw0nlqxd83agj-extra-cmake-modules-6.14.0/share/ECM/modules/ECMFindQmlModule.cmake:56 (find_package)
  dashboard/CMakeLists.txt:7 (ecm_find_qmlmodule)


-- Could NOT find org.kde.kirigami-QMLModule (missing: org.kde.kirigami-QMLModule_FOUND)
CMake Warning (dev) at /nix/store/8qs879yx84gng0yljdybwi4fbbgg3dyy-qtbase-6.9.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:3392 (message):
  Qt policy QTP0004 is not set: You need qmldir files for each extra
  directory that contains .qml files for your module.  Check
  https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html for policy details.
  Use the qt_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/smz7rlw4r4p5bqngzkb1svqb3m5gvc2w-qtdeclarative-6.9.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:3503 (__qt_internal_setup_policy)
  /nix/store/smz7rlw4r4p5bqngzkb1svqb3m5gvc2w-qtdeclarative-6.9.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:916 (qt6_target_qml_sources)
  /nix/store/smz7rlw4r4p5bqngzkb1svqb3m5gvc2w-qtdeclarative-6.9.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:1252 (qt6_add_qml_module)
  dashboard/CMakeLists.txt:28 (qt_add_qml_module)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Configuring done (11.8s)
-- Generating done (0.5s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    CUDA_TOOLKIT_ROOT_DIR
    FETCHCONTENT_FULLY_DISCONNECTED
    QT_OPTIONAL_TOOLS_PATH


-- Build files have been written to: /build/source/build
cmake: enabled parallel building
cmake: enabled parallel installing
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: -j12 SHELL=/nix/store/xy4jjgw87sbgwylm5kn047d9gkbhsr9x-bash-5.2p37/bin/bash
[  1%] Generating b_generated_bindings.c
[  1%] Built target aux_os_autogen_timestamp_deps
[  1%] Built target wivrn-common-base_autogen_timestamp_deps
[  1%] Generating wivrn_server_dbus.c, wivrn_server_dbus.h
[  1%] Copying wivrn-dashboard qml resources into build dir
[  1%] Generating nvpro_pyramid/srgba8_mipmap_fast_pipeline.comp.h
[  3%] Generating share/locale/es/LC_MESSAGES/wivrn-dashboard.mo
[  3%] Built target aux_util_debug_gui_autogen_timestamp_deps
[  3%] Built target xrt-external-nanopb_autogen_timestamp_deps
[  3%] Built target wivrnctl_autogen_timestamp_deps
[  4%] Copying wivrn-dashboard qml sources into build dir
[  4%] Generating share/locale/fr/LC_MESSAGES/wivrn-dashboard.mo
[  4%] Built target aux_math_autogen_timestamp_deps
[  4%] Built target wivrn-dashboard_copy_res
[  6%] Generating nvpro_pyramid/srgba8_mipmap_general_pipeline.comp.h
[  6%] Generating b_generated_bindings.h
[  7%] Automatic MOC and UIC for target aux_os
[  7%] Built target wivrn-dashboard_copy_qml
[  7%] Generating share/locale/pt_BR/LC_MESSAGES/wivrn-dashboard.mo
[  7%] Generating share/locale/zh_TW/LC_MESSAGES/wivrn-dashboard.mo
[  7%] Automatic MOC and UIC for target aux_util_debug_gui
[  9%] Automatic MOC and UIC for target wivrn-common-base
[  9%] Automatic MOC and UIC for target wivrnctl
[  9%] Automatic MOC and UIC for target xrt-external-nanopb
[  9%] Built target wivrn-dashboard-translations
[  9%] Automatic MOC and UIC for target aux_math
[  9%] Built target aux_os_autogen
[  9%] Built target aux_util_debug_gui_autogen
[  9%] Built target wivrn-common-base_autogen
[  9%] Built target aux_generated_bindings_autogen_timestamp_deps
[  9%] Building CXX object _deps/monado-build/src/xrt/auxiliary/os/CMakeFiles/aux_os.dir/aux_os_autogen/mocs_compilation.cpp.o
[  9%] Building C object _deps/monado-build/src/xrt/auxiliary/os/CMakeFiles/aux_os.dir/os_hid_hidraw.c.o
[  9%] Built target xrt-external-nanopb_autogen
[  9%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util_debug_gui.dir/aux_util_debug_gui_autogen/mocs_co>
[  9%] Building CXX object _deps/monado-build/src/xrt/auxiliary/os/CMakeFiles/aux_os.dir/os_time.cpp.o
[  9%] Built target wivrnctl_autogen
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/external/nvpro_pyramid/srgba8_mipmap_general_pipeline.comp
[  9%] Building CXX object common/CMakeFiles/wivrn-common-base.dir/wivrn-common-base_autogen/mocs_compilation.cpp.o
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/external/nvpro_pyramid/srgba8_mipmap_fast_pipeline.comp
[  9%] Built target wivrn-server-dbus-codegen
[  9%] Automatic MOC and UIC for target aux_generated_bindings
[  9%] Building CXX object _deps/monado-build/src/external/CMakeFiles/xrt-external-nanopb.dir/xrt-external-nanopb_autogen/mocs_compilatio>
[ 10%] Building CXX object common/CMakeFiles/wivrn-common-base.dir/utils/xdg_base_directory.cpp.o
[ 10%] Built target aux_math_autogen
[ 10%] Built target xrt-external-nvpro-pyramid
[ 10%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util_debug_gui.dir/u_debug_gui.c.o
[ 10%] Building CXX object common/CMakeFiles/wivrn-common-base.dir/utils/flatpak.cpp.o
[ 10%] Building C object _deps/monado-build/src/external/CMakeFiles/xrt-external-nanopb.dir/nanopb/monado_metrics.pb.c.o
[ 10%] Building CXX object common/CMakeFiles/wivrn-common-base.dir/utils/strings.cpp.o
[ 10%] Building CXX object common/CMakeFiles/wivrn-common-base.dir/version.cpp.o
[ 12%] Building CXX object tools/wivrnctl/CMakeFiles/wivrnctl.dir/wivrnctl_autogen/mocs_compilation.cpp.o
[ 12%] Built target wivrn-server-dbus_autogen_timestamp_deps
[ 12%] Building C object _deps/monado-build/src/external/CMakeFiles/xrt-external-nanopb.dir/nanopb/pb_common.c.o
[ 14%] Building C object _deps/monado-build/src/external/CMakeFiles/xrt-external-nanopb.dir/nanopb/pb_decode.c.o
[ 14%] Building CXX object tools/wivrnctl/CMakeFiles/wivrnctl.dir/main.cpp.o
[ 14%] Building C object _deps/monado-build/src/external/CMakeFiles/xrt-external-nanopb.dir/nanopb/pb_encode.c.o
[ 14%] Automatic MOC and UIC for target wivrn-server-dbus
[ 14%] Built target aux_generated_bindings_autogen
[ 14%] Linking CXX static library libaux_util_debug_gui.a
[ 14%] Built target aux_util_debug_gui
[ 14%] Built target wivrn-server-dbus_autogen
[ 15%] Building CXX object server/CMakeFiles/wivrn-server-dbus.dir/wivrn-server-dbus_autogen/mocs_compilation.cpp.o
[ 15%] Building C object server/CMakeFiles/wivrn-server-dbus.dir/wivrn_server_dbus.c.o
[ 15%] Linking CXX static library libaux_os.a
[ 15%] Built target aux_os
[ 15%] Building C object _deps/monado-build/src/xrt/auxiliary/bindings/CMakeFiles/aux_generated_bindings.dir/b_generated_bindings.c.o
[ 15%] Building CXX object _deps/monado-build/src/xrt/auxiliary/bindings/CMakeFiles/aux_generated_bindings.dir/aux_generated_bindings_aut>
[ 15%] Linking CXX static library libxrt-external-nanopb.a
[ 15%] Built target xrt-external-nanopb
[ 15%] Linking CXX static library libwivrn-server-dbus.a
[ 15%] Built target wivrn-server-dbus
[ 15%] Linking CXX static library libwivrn-common-base.a
[ 15%] Built target wivrn-common-base
[ 15%] Built target wivrn-common_autogen_timestamp_deps
[ 15%] Built target wivrn-dashboard_autogen_timestamp_deps
[ 15%] Automatic MOC and UIC for target wivrn-common
[ 15%] Automatic MOC and UIC for target wivrn-dashboard
[ 15%] Built target wivrn-common_autogen
[ 15%] Building CXX object common/CMakeFiles/wivrn-common.dir/crypto.cpp.o
[ 17%] Building CXX object common/CMakeFiles/wivrn-common.dir/smp.cpp.o
[ 17%] Building CXX object common/CMakeFiles/wivrn-common.dir/wivrn-common_autogen/mocs_compilation.cpp.o
[ 17%] Building CXX object common/CMakeFiles/wivrn-common.dir/wivrn_sockets.cpp.o
[ 17%] Building CXX object common/CMakeFiles/wivrn-common.dir/secrets.cpp.o
[ 17%] Building CXX object common/CMakeFiles/wivrn-common.dir/utils/xdg_base_directory.cpp.o
[ 18%] Building CXX object common/CMakeFiles/wivrn-common.dir/utils/strings.cpp.o
[ 18%] Building CXX object common/CMakeFiles/wivrn-common.dir/vk/allocation.cpp.o
[ 18%] Building CXX object common/CMakeFiles/wivrn-common.dir/vk/error_category.cpp.o
[ 18%] Building CXX object common/CMakeFiles/wivrn-common.dir/vk/vk_allocator.cpp.o
[ 18%] Building CXX object common/CMakeFiles/wivrn-common.dir/vk/vk_mem_alloc.cpp.o
[ 18%] Linking CXX static library libaux_generated_bindings.a
[ 18%] Built target aux_generated_bindings
[ 18%] Built target aux_util_autogen_timestamp_deps
[ 20%] Automatic MOC and UIC for target aux_util
[ 20%] Built target aux_util_autogen
[ 21%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/aux_util_autogen/mocs_compilation.cpp.o
[ 21%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_bitwise.c.o
[ 21%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_autoexpgain.c.o
[ 21%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_builders.c.o
[ 21%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_debug.c.o
[ 21%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_deque.cpp.o
[ 23%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_device.c.o
[ 23%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_distortion.c.o
[ 23%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_distortion_mesh.c.o
[ 23%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_file.c.o
[ 23%] Built target wivrn-dashboard_autogen
[ 23%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_file.cpp.o
[ 23%] Generating wivrn_server_dbus.cpp, wivrn_server_dbus.h
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 25%] Running AUTOMOC file extraction for target wivrn-dashboard
[ 26%] Running rcc for resource qmake_io_github_wivrn_wivrn
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 28%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_format.c.o
[ 28%] Running rcc for resource wivrn-dashboard_raw_qml_0
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 28%] Generating .rcc/qmlcache/wivrn-dashboard_qmlcache_loader.cpp
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 28%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_frame.c.o
[ 28%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_hand_tracking.c.o
[ 28%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_hand_simulation.c.o
[ 28%] Running moc --collect-json for target wivrn-dashboard
[ 28%] Generating moc_wivrn_server_dbus.cpp
[ 28%] Generating meta_types/qt6wivrn-dashboard_release_metatypes.json
[ 28%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_handles.c.o
[ 28%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_hashmap.cpp.o
[ 28%] Automatic QML type registration for target wivrn-dashboard
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 29%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_hashset.cpp.o
[ 29%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_id_ringbuffer.cpp.o
Warning: steam_apps.h:: steamApp is registered multiple times by the following C++ classes: steamApp, steamApp
Warning: wivrn_server.h:: headset is registered multiple times by the following C++ classes: headset, headset
[ 29%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_imu_sink_split.c.o
[ 29%] Generating .rcc/qmlcache/wivrn-dashboard_qml/ApkInstallPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/ApkInstallPage_qml.cpp.aots>
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 29%] Generating .rcc/qmlcache/wivrn-dashboard_qml/BetterLabel_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/BetterLabel_qml.cpp.aotstats
[ 31%] Generating .rcc/qmlcache/wivrn-dashboard_qml/ConnectUsbDialog_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/ConnectUsbDialog_qml.cpp.>
[ 31%] Generating .rcc/qmlcache/wivrn-dashboard_qml/DeviceCard_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/DeviceCard_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 31%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_imu_sink_force_monotonic.c.o
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 32%] Generating .rcc/qmlcache/wivrn-dashboard_qml/HeadsetStatsPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/HeadsetStatsPage_qml.cpp.>
[ 32%] Generating .rcc/qmlcache/wivrn-dashboard_qml/HeadsetsPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/HeadsetsPage_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 32%] Generating .rcc/qmlcache/wivrn-dashboard_qml/Main_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/Main_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 32%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_json.c.o
[ 32%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_limited_unique_id.cpp.o
[ 34%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_live_stats.cpp.o
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/SelectGame_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/SelectGame_qml.cpp.aotstats
[ 34%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_logging.c.o
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 34%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_metrics.c.o
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/SettingsPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/SettingsPage_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/SteamLaunchOptions_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/SteamLaunchOptions_qml.>
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 34%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_misc.c.o
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/TroubleshootCard_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/TroubleshootCard_qml.cpp.>
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/TroubleshootPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/TroubleshootPage_qml.cpp.>
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/WizardPage_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/WizardPage_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 34%] Generating .rcc/qmlcache/wivrn-dashboard_qml/WizardStep_qml.cpp, .rcc/qmlcache/wivrn-dashboard_qml/WizardStep_qml.cpp.aotstats
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_pacing_compositor.c.o
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_pacing_app.c.o
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_pacing_compositor_fake.c.o
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_pretty_print.c.o
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_prober.c.o
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_session.c.o
[ 35%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/wivrn-dashboard_autogen/mocs_compilation.cpp.o
[ 35%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_space_overseer.c.o
[ 37%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_string_list.cpp.o
[ 37%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/main.cpp.o
[ 37%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/wivrn_server.cpp.o
[ 37%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_system.c.o
[ 37%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_system_helpers.c.o
[ 39%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/wivrn_qdbus_types.cpp.o
[ 39%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/steam_apps.cpp.o
[ 39%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/escape_string.cpp.o
[ 39%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_time.cpp.o
[ 39%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/settings.cpp.o
[ 39%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/rectangle_partitionner.cpp.o
[ 39%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_trace_marker.c.o
[ 40%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_tracked_imu_3dof.c.o
[ 40%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_var.cpp.o
[ 40%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_vector.cpp.o
[ 40%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/adb.cpp.o
[ 40%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_config_json.c.o
[ 42%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/apk_installer.cpp.o
[ 42%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_visibility_mask.c.o
[ 42%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_win32_com_guard.cpp.o
[ 43%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_worker.c.o
[ 43%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_worker.cpp.o
[ 43%] Linking CXX static library libwivrn-common.a
[ 43%] Built target wivrn-common
[ 43%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/vulkan_info.cpp.o
[ 43%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/wivrn_server_dbus.cpp.o
[ 43%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_git_tag.c.o
[ 43%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util.dir/u_linux.c.o
[ 43%] Linking CXX static library libaux_util.a
[ 43%] Built target aux_util
[ 43%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/aux_math_autogen/mocs_compilation.cpp.o
[ 43%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_base.cpp.o
[ 43%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_clock_tracking.c.o
[ 45%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_filter_fifo.c.o
[ 45%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/wivrn-dashboard_qmltyperegistrations.cpp.o
[ 45%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.qt/rcc/qrc_qmake_io_github_wivrn_wivrn.cpp.o
[ 46%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qmlcache_loader.cpp.o
[ 46%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/Main_qml.cpp.o
[ 46%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_filter_one_euro.c.o
[ 46%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_hash.cpp.o
[ 46%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_imu_3dof.c.o
[ 46%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_imu_pre.c.o
[ 46%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/ApkInstallPage_qml.cpp.o
[ 48%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_lowpass_float.cpp.o
[ 48%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_lowpass_integer.cpp.o
[ 48%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_matrix_projection.cpp.o
[ 48%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/SettingsPage_qml.cpp.o
[ 48%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_optics.c.o
[ 48%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/HeadsetsPage_qml.cpp.o
[ 48%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/HeadsetStatsPage_qml.cpp.o
[ 48%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_permutation.c.o
[ 50%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/WizardPage_qml.cpp.o
[ 50%] Building C object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_predict.c.o
[ 50%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/WizardStep_qml.cpp.o
[ 50%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/DeviceCard_qml.cpp.o
[ 50%] Linking CXX executable ../../server/wivrnctl
[ 50%] Built target wivrnctl
[ 50%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/BetterLabel_qml.cpp.o
[ 50%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/ConnectUsbDialog_qml.cpp.o
[ 51%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_quatexpmap.cpp.o
[ 53%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/SelectGame_qml.cpp.o
[ 53%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/SteamLaunchOptions_qml.cpp.o
[ 53%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_relation_history.cpp.o
[ 53%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/TroubleshootPage_qml.cpp.o
[ 53%] Building CXX object _deps/monado-build/src/xrt/auxiliary/math/CMakeFiles/aux_math.dir/m_space.cpp.o
[ 53%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.rcc/qmlcache/wivrn-dashboard_qml/TroubleshootCard_qml.cpp.o
[ 53%] Building CXX object dashboard/CMakeFiles/wivrn-dashboard.dir/.qt/rcc/qrc_wivrn-dashboard_raw_qml_0.cpp.o
[ 53%] Linking CXX static library libaux_math.a
[ 53%] Built target aux_math
[ 53%] Built target aux_vk_autogen_timestamp_deps
[ 53%] Generating ipc_protocol_generated.h from protocol JSON description
[ 53%] Built target aux_ogl_autogen_timestamp_deps
[ 53%] Built target aux_util_process_autogen_timestamp_deps
[ 53%] Built target aux_tracking_autogen_timestamp_deps
[ 53%] Built target comp_multi_autogen_timestamp_deps
[ 53%] Automatic MOC and UIC for target aux_vk
[ 54%] Automatic MOC and UIC for target aux_ogl
[ 54%] Automatic MOC and UIC for target aux_util_process
[ 54%] Automatic MOC and UIC for target aux_tracking
[ 54%] Automatic MOC and UIC for target comp_multi
[ 54%] Built target ipc_shared_autogen_timestamp_deps
[ 54%] Built target aux_ogl_autogen
[ 54%] Built target aux_vk_autogen
[ 54%] Built target aux_util_process_autogen
[ 56%] Building CXX object _deps/monado-build/src/xrt/auxiliary/ogl/CMakeFiles/aux_ogl.dir/aux_ogl_autogen/mocs_compilation.cpp.o
[ 56%] Automatic MOC and UIC for target ipc_shared
[ 56%] Built target aux_tracking_autogen
[ 56%] Built target comp_multi_autogen
[ 56%] Building C object _deps/monado-build/src/xrt/auxiliary/ogl/CMakeFiles/aux_ogl.dir/ogl_api.c.o
[ 56%] Building CXX object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/aux_vk_autogen/mocs_compilation.cpp.o
[ 56%] Building C object _deps/monado-build/src/xrt/auxiliary/ogl/CMakeFiles/aux_ogl.dir/ogl_helpers.c.o
[ 56%] Building C object _deps/monado-build/src/xrt/auxiliary/ogl/CMakeFiles/aux_ogl.dir/glx_api.c.o
[ 56%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_bundle_init.c.o
[ 56%] Building C object _deps/monado-build/src/xrt/auxiliary/ogl/CMakeFiles/aux_ogl.dir/egl_api.c.o
[ 56%] Built target ipc_shared_autogen
[ 56%] Building CXX object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util_process.dir/aux_util_process_autogen/mocs_compil>
[ 57%] Building C object _deps/monado-build/src/xrt/auxiliary/util/CMakeFiles/aux_util_process.dir/u_process.c.o
[ 57%] Building CXX object _deps/monado-build/src/xrt/auxiliary/tracking/CMakeFiles/aux_tracking.dir/aux_tracking_autogen/mocs_compilatio>
[ 59%] Building C object _deps/monado-build/src/xrt/auxiliary/tracking/CMakeFiles/aux_tracking.dir/t_data_utils.c.o
[ 59%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_cmd.c.o
[ 59%] Linking CXX static library libaux_util_process.a
[ 59%] Built target aux_util_process
[ 59%] Building CXX object _deps/monado-build/src/xrt/auxiliary/tracking/CMakeFiles/aux_tracking.dir/t_imu.cpp.o
[ 60%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_cmd_pool.c.o
[ 62%] Building CXX object _deps/monado-build/src/xrt/compositor/CMakeFiles/comp_multi.dir/comp_multi_autogen/mocs_compilation.cpp.o
[ 62%] Building CXX object _deps/monado-build/src/xrt/auxiliary/tracking/CMakeFiles/aux_tracking.dir/t_openvr_tracker.cpp.o
[ 62%] Building C object _deps/monado-build/src/xrt/compositor/CMakeFiles/comp_multi.dir/multi/comp_multi_compositor.c.o
[ 62%] Linking CXX static library libaux_ogl.a
[ 62%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_compositor_flags.c.o
[ 62%] Built target aux_ogl
[ 62%] Building C object _deps/monado-build/src/xrt/compositor/CMakeFiles/comp_multi.dir/multi/comp_multi_system.c.o
[ 62%] Building CXX object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_shared.dir/ipc_shared_autogen/mocs_compilation.cpp.o
[ 62%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_shared.dir/shared/ipc_shmem.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_shared.dir/shared/ipc_utils.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_debug.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_enumerate.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_shared.dir/shared/ipc_message_channel_unix.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_function_loaders.c.o
[ 62%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_helpers.c.o
[ 64%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_image_allocator.c.o
[ 65%] Linking CXX static library libipc_shared.a
[ 65%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_image_readback_to_xf_pool.c.o
[ 65%] Built target ipc_shared
[ 65%] Linking CXX static library libcomp_multi.a
[ 65%] Generating ipc_client_generated.c from protocol JSON description
[ 65%] Built target comp_multi
[ 65%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_print.c.o
[ 65%] Generating ipc_client_generated.h from protocol JSON description
[ 67%] Generating ipc_server_generated.c from protocol JSON description
[ 67%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_state_creators.c.o
[ 67%] Built target ipc_client_autogen_timestamp_deps
[ 67%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_surface_info.c.o
[ 67%] Generating ipc_server_generated.h from protocol JSON description
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c: In function 'vk_object_type_string':
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:251:29: error: 'VK_OBJECT_TYPE_CUDA_MODULE_N>
  251 |                 ENUM_TO_STR(VK_OBJECT_TYPE_CUDA_MODULE_NV);
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:54:14: note: in definition of macro 'ENUM_TO>
   54 |         case r: return #r
      |              ^
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:251:29: note: each undeclared identifier is >
  251 |                 ENUM_TO_STR(VK_OBJECT_TYPE_CUDA_MODULE_NV);
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:54:14: note: in definition of macro 'ENUM_TO>
   54 |         case r: return #r
      |              ^
[ 67%] Built target ipc_server_autogen_timestamp_deps
[ 68%] Automatic MOC and UIC for target ipc_client
[ 68%] Building C object _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/vk_sync_objects.c.o
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:252:29: error: 'VK_OBJECT_TYPE_CUDA_FUNCTION>
  252 |                 ENUM_TO_STR(VK_OBJECT_TYPE_CUDA_FUNCTION_NV);
      |                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/5yv9r1hx6nn0h7pw0z2n076s3dzxsdbn-source-patched/src/xrt/auxiliary/vk/vk_helpers.c:54:14: note: in definition of macro 'ENUM_TO>
   54 |         case r: return #r
      |              ^
make[2]: *** [_deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/build.make:191: _deps/monado-build/src/xrt/auxiliary/vk/CMake>
make[2]: *** Waiting for unfinished jobs....
[ 68%] Automatic MOC and UIC for target ipc_server
[ 68%] Built target ipc_client_autogen
[ 68%] Building CXX object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/ipc_client_autogen/mocs_compilation.cpp.o
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/ipc_client_generated.c.o
[ 70%] Built target ipc_server_autogen
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_compositor.c.o
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_connection.c.o
[ 70%] Building CXX object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/ipc_server_autogen/mocs_compilation.cpp.o
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_device.c.o
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_hmd.c.o
[ 70%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/ipc_server_generated.c.o
[ 71%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_instance.c.o
make[1]: *** [CMakeFiles/Makefile2:3022: _deps/monado-build/src/xrt/auxiliary/vk/CMakeFiles/aux_vk.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/server/ipc_server_handler.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_session.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_space_overseer.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/server/ipc_server_per_client_thread.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/server/ipc_server_process.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_system.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_server.dir/server/ipc_server_mainloop_linux.c.o
[ 73%] Building C object _deps/monado-build/src/xrt/ipc/CMakeFiles/ipc_client.dir/client/ipc_client_system_devices.c.o
[ 73%] Linking CXX static library libipc_client.a
[ 73%] Built target ipc_client
[ 73%] Linking CXX executable ../server/wivrn-dashboard
[ 73%] Linking CXX static library libipc_server.a
[ 73%] Built target ipc_server
[ 73%] Built target wivrn-dashboard
[ 73%] Linking CXX static library libaux_tracking.a
[ 73%] Built target aux_tracking
make: *** [Makefile:136: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.28, NixOS, 25.05 (Warbler), 25.05.20250513.fab95ba`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.3`
 - nixpkgs: `/nix/store/bk3n875j54pdcxhp8k2lhm738n9b680z-source`

### Notify maintainers

@PassiveLemon

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/wi/wivrn/package.nix
+++ b/pkgs/by-name/wi/wivrn/package.nix
@@ -196,14 +196,14 @@ stdenv.mkDerivation (finalAttrs: {
 
   passthru.updateScript = nix-update-script { };
 
-  meta = with lib; {
+  meta = {
     description = "An OpenXR streaming application to a standalone headset";
     homepage = "https://github.com/WiVRn/WiVRn/";
     changelog = "https://github.com/WiVRn/WiVRn/releases/";
-    license = licenses.gpl3Only;
-    maintainers = with maintainers; [ passivelemon ];
-    platforms = platforms.linux;
+    license = lib.licenses.gpl3Only;
+    maintainers = [ lib.maintainers.passivelemon ];
+    platforms = lib.platforms.linux;
     mainProgram = "wivrn-server";
-    sourceProvenance = with sourceTypes; [ fromSource ];
+    sourceProvenance = [ lib.sourceTypes.fromSource ];
   };
 })

--- a/pkgs/by-name/wi/wivrn/package.nix
+++ b/pkgs/by-name/wi/wivrn/package.nix
@@ -4,6 +4,7 @@
   stdenv,
   fetchFromGitHub,
   fetchFromGitLab,
+  fetchpatch2,
   applyPatches,
   autoAddDriverRunpath,
   avahi,
@@ -71,6 +72,10 @@ stdenv.mkDerivation (finalAttrs: {
 
     patches = [
       ./force-enable-steamvr_lh.patch
+      (fetchpatch2 {
+        url = "https://gitlab.freedesktop.org/monado/monado/-/commit/2a6932d46dad9aa957205e8a47ec2baa33041076.patch";
+        hash = "sha256-CZMbGgx7mEDcjcoRJHDZ5P6BecFW8CB4fpzxQ9bpAvE=";
+      })
     ];
 
     postPatch = ''

